azurelinuxagent/ga/exthandlers.py

```diff
@@ -38,7 +38,7 @@
 from azurelinuxagent.common.errorstate import ErrorState, ERROR_STATE_DELTA
 
 from azurelinuxagent.common.event import add_event, WALAEventOperation, elapsed_milliseconds
-from azurelinuxagent.common.exception import ExtensionError, ProtocolError, RestartError
+from azurelinuxagent.common.exception import ExtensionError, ProtocolError, HttpError
```

Comment:
It looks like `HttpError` is imported here but not used anywhere within this file.